packages/@react-spectrum/s2/stories/Accordion.stories.tsx

```diff
@@ -130,6 +131,68 @@ WithDisabledDisclosure.parameters = {
   }
 };
 
+function ControlledAccordion(props) {
+  let [expandedKeys, setExpandedKeys] = React.useState<Set<Key>>(new Set(['people']));
+  return (
+    <div className={style({font: 'body', display: 'flex', flexDirection: 'column', gap: 8})}>
+      <Accordion
+        onExpandedChange={setExpandedKeys}
+        expandedKeys={expandedKeys}
+        {...props}>
+        <Disclosure id="files">
+          <DisclosureHeader>
```

Comment:
these need to be `DisclosureTitle` instead of `DisclosureHeader` now. same thing in the Disclosure stories too